src/phantom.js

```diff
@@ -30,8 +31,9 @@ export default class Phantom {
             throw new Error('Unexpected type of parameters. Expecting args to be array.');
         }
 
-        logger.debug(`Starting ${phantomjs.path} ${args.concat([__dirname + '/shim.js']).join(' ')}`);
-        this.process = spawn(phantomjs.path, args.concat([__dirname + '/shim.js']));
+        let pathToShim = path.normalize(__dirname + '/shim.js');
```

Comment:
the pathing breaks the tests on windows systems in the tests, without normalizing you wind up with / and \ in the path as opposed to just \ (again only on windows) seamed lie the best solution to normalize the path here as normalizing cleans up the path anyway